change/@ni-nimble-components-2f1afb28-1f86-43f7-9618-d0e54d7221e0.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Curious for your thoughts on auto testing this. The [select theme matrix](https://60e89457a987cf003efc0a5b-atkkxnkzpr.chromatic.com/?path=/story/tests-select--select-theme-matrix) doesn't include placeholders but we have [a story](https://60e89457a987cf003efc0a5b-atkkxnkzpr.chromatic.com/?path=/story/tests-select--select-below-open-no-filter-light-theme-white-background-with-placeholder) for each theme with the value set to a placeholder. Would it be easy to add a second select to each of those stories that's disabled?

Reply:
Probably would rather add the case to the matrix story. I considered the fact that this had been a visual test gap (was kind of surprised that no visual tests had diffs), but decided it wasn't really that important. But since you brought it up, I'll go ahead and add a test case.

Reply:
Actually, the select theme matrix story was taking a long time to load, so I moved the "required asterisk" testing into a separate story and created a separate story for placeholder text.

Reply:
Reminder to update the PR description Testing section